src/components/Hero/Hero.scss

```diff
@@ -0,0 +1,153 @@
+@import "../../sass/_variables.scss";
+
+.hero-wrap {
```

Comment:
**Note to reviewers**: Taken from [edx-mktg/docroot/sites/all/themes/atedx/sass/elements/_masters-product-page.scss](https://github.com/edx/edx-mktg/blob/a3a9a79e229d83f7eb0099db02aa48bdee7dbe3e/docroot/sites/all/themes/atedx/sass/elements/_masters-product-page.scss#L14)